bugwarrior/services/taiga.py

```diff
@@ -131,3 +150,15 @@ def annotations(self, story, project):
             ) for item in history if item['comment']),
             self.build_url(story, project)
         )
+
+    def annotations_tasks(self, task, project):
+        url = '%s/api/v1/history/task/%i' % (self.url, task['id'])
+        response = self.session.get(url)
+        history = response.json()
+        return self.build_annotations(
+            ((
+                item['user']['username'],
+                item['comment'],
+            ) for item in history if item['comment']),
+            self.build_url_tasks(task, project)
+        )
```

Comment:
Couldn't we keep the single `annotations` and `build_url` methods but add a `task_type` argument? E.g.:

```py
def build_url(self, story, project, task_type):
    return '%s/project/%s/%s/%i' % (self.url, project['slug'], task_type, task['ref'])

def annotations(self, task, project, task_type):
    url = '%s/api/v1/history/%s/%i' % (self.url, task_type, task['id'])
    ...
        self.build_url(task, project, task_type)
    ...
```